src/index.html

```diff
@@ -6,22 +6,15 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>CodeMirror 6 + Perspective</title>
 
-  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective@2.10.1/dist/cdn/perspective.js"></script>
-  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer@2.10.1/dist/cdn/perspective-viewer.js"></script>
-  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer-datagrid@2.10.1/dist/cdn/perspective-viewer-datagrid.js"></script>
-  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer-d3fc@2.10.1/dist/cdn/perspective-viewer-d3fc.js"></script>
-
-  <link rel="stylesheet" crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer/dist/css/themes.css" />
-
-  <link rel="preload" href="https://cdn.jsdelivr.net/npm/@finos/perspective@2.10.1/dist/cdn/perspective.cpp.wasm" as="fetch" type="application/wasm" crossorigin="anonymous" />
-  <link rel="preload" href="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer@2.10.1/dist/cdn/perspective_bg.wasm" as="fetch" type="application/wasm" crossorigin="anonymous" />
-  <link rel="preload" href="https://cdn.jsdelivr.net/npm/superstore-arrow/superstore.lz4.arrow" as="fetch" type="arraybuffer" crossorigin="anonymous" />
-  <link rel="preload" href="https://cdn.jsdelivr.net/npm/@finos/perspective@2.10.1/dist/cdn/perspective.worker.js" as="fetch" type="application/javascript" crossorigin="anonymous" />
-
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective@3.0.0-rc.1/dist/cdn/perspective.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer@3.0.0-rc.1/dist/cdn/perspective-viewer.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer-datagrid@3.0.0-rc.1/dist/cdn/perspective-viewer-datagrid.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer-d3fc@3.0.0-rc.1/dist/cdn/perspective-viewer-d3fc.js"></script>
+  <link rel="stylesheet" crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/@finos/perspective-viewer@3.0.0-rc.1/dist/css/themes.css" />
   <script type="module">
-    import { worker } from "https://cdn.jsdelivr.net/npm/@finos/perspective@2.10.1/dist/cdn/perspective.js";
+    import { worker } from "https://cdn.jsdelivr.net/npm/@finos/perspective@3.0.0-rc.1/dist/cdn/perspective.js";
 
-    const WORKER = worker();
+    const WORKER = await worker();
```

Comment:
FWIW I pulled this from the [editable example](https://perspective.finos.org/block/?example=editable) so may want to add the `await` there too.